docs/user-guide/patterns/sums.md

```diff
@@ -1,14 +1,30 @@
-Tracking Sums
-=============
+# Tracking Sums
 
-```{todo}
-Finish
+## Enforcing Sum of Two Balances Constraint
+```cvl
+invariant directSumOfTwo(address a, address b)
+    (a != b) => (balanceOf(a) + balanceOf(b) <= to_mathint(totalSupply()));
 ```
 
-```{todo}
-Sum of two values is ≤ total
-```
+Ensure that the sum of balances for any two distinct addresses, `a` and `b`, does not exceed 
+the total supply.
+
+## Maintaining Equality Between Sum of Balances and Total Supply
+```cvl
+ghost mathint sumBalances {
+    init_state axiom sumBalances == 0;
+}
 
-```{todo}
-Sum is sum of all values
+hook Sstore balanceOf[KEY address user] uint256 newBalance (uint256 oldBalance) STORAGE
+{
+    # ghost variables do not accept += operator
+    sumBalances = sumBalances + newBalance - oldBalance;
+}
+
+invariant totalIsSumBalances()
+    to_mathint(totalSupply()) == sumBalances;
 ```
+
+Track the sum of all balances and ensure that it remains equal to the total supply. The 
+`sumBalances` ghost variable is updated with changes in individual balances using a storage 
+hook, ensuring accuracy and consistency in the overall sum.
```

Comment:
todo, maybe make a ticket - show this on a real contract and common pitfalls. sometimes we need to add additional requires because the sum ghost is not really representing the sum, it just updates the same way as the real sum

Reply:
Added a ref to the tutorial page elaborate on the topic